### PR TITLE
WRF-Hydro v5.0.0 release changes

### PR DESCRIPTION
**Pull request by kafitzgerald**
Changes:
* Update code version for output file metadata
* Remove beta warning from README
* Add WRF-Hydro logo to README
* Point to WRF-Hydro website for documentation
* Add README for testing

### CODE REVIEW

File: tests/testing_readme.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# To run the tests included in the wrf_hydro_nwm_public/tests directory:
```
Comment (jmccreight) [inline tests/testing_readme.txt]: I thought this file was to be removed. There is a README.md in tests/ now.

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# WRF-Hydro v5.0.0
+
+## High-Level Highlights:
+- New capability to aggregate and route flow with user-defined mapping (catchments and reaches, e.g., NHDPlus).
+- New capability to specify key hydrologic and vegetation parameters in 2 dimensions (and 3 in the case of soil properties).
+- New Noah-MP surface resistance formulation that improves snowpack simulation.
+- Updates to Noah-MP infiltration scheme to better handle high intensity rainfall events in fast-draining soils.
+- Significant improvements to model output routines, including full CF compliance, new capabilities for applying internal scale/offset and compression to reduce file sizes, and built-in coordinate information to allow outputs to be read natively in GIS environments.
+- New streamflow nudging data-assimilation capability for the Muskingum-Cunge method.
+
```
Comment (jmccreight) [inline NEWS.md]: Add a high-level bullet about testing available in the tests/ directory with details there?

Comment (mollymca) [reply NEWS.md]: Is testing available to the public?

Comment (mollymca) [reply NEWS.md]: Do we need to add that to the full release notes?

Comment (jmccreight) [reply NEWS.md]: Yes and yes. One line should suffice on its existence and availablility

**Pull request merged**